### PR TITLE
Make TagComponent networked

### PR DESCRIPTION
**Pull request by PixelTheKermit**
## About the PR

Made TagsComponent actually network properly to the client.

## Why / Balance

Need this downstream.

## Technical details

TagsComponent was modified slightly. Otherwise no further changes were made.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Left is server, right is client. (idk if there's any other way of showing this.)
image

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Shared/Tag/TagComponent.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+﻿using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
 
 namespace Content.Shared.Tag
 {
-    [RegisterComponent, Access(typeof(TagSystem))]
+    [RegisterComponent, NetworkedComponent, Access(typeof(TagSystem))]
     public sealed partial class TagComponent : Component
     {
+        [AutoNetworkedField]
```
Comment (metalgearsloth) [inline Content.Shared/Tag/TagComponent.cs:9]: This is doing nothing please remove.

Comment (PixelTheKermit) [reply Content.Shared/Tag/TagComponent.cs:9]: Gonna remove and test, I'll be back in a bit.

Comment (PixelTheKermit) [reply Content.Shared/Tag/TagComponent.cs]: Should be resolved now.

**Comment by PixelTheKermit** [comment]
The tests failing seem to be audio related. Not sure if that's my problem to resolve or not.

**Pull request merged**